File: PSW-projekat/hospital-be-main/src/HospitalLibrary/Core/Model/Therapy.cs
namespace HospitalLibrary.Core.Model
{
    public class Therapy: BaseModel
    {
        public virtual Medicine Medicine { get; set; }
        public virtual Blood Blood { get; set; }
        public int QuantitytMedicine { get; set; }
        public int QuantityBlood { get; set; }

        public Therapy() { }
        public Therapy(Medicine medicine, Blood blood, int quantitytMedicine, int quantityBlood)
        {
            Medicine = medicine;
            Blood = blood;
            QuantitytMedicine = quantitytMedicine;
            QuantityBlood = quantityBlood;
        }
    }
}

File: PSW-projekat/PZL-Angular/src/app/modules/hospital/model/therapy.ts
import { Blood } from "./blood";
import { Medicine } from "./medicine";

export class Therapy {
    id: number;
    deleted: boolean;
    medicine: Medicine;
    blood: Blood;
    quantitytMedicine: number = 0;
    quantityBlood: number = 0;

    public constructor(id: any, deleted: any, medicine: any, blood: any, quantitytMedicine: any, quantityBlood: any ) {
        this.id = id;
        this.deleted = deleted;
        this.medicine = medicine;
        this.blood = blood;
        this.quantitytMedicine = quantitytMedicine;
        this.quantityBlood =quantityBlood;

    }
}